testdata/scripts/tiny.txt

```diff
@@ -1,43 +1,32 @@
 env GOPRIVATE=test/main
 
-env TINY_PATTERN='^\/\/line :1$'
-env DEFAULT_PATTERN='^\/\/line \w\.go:[1-9][0-9]*$'
-env DEFAULT_STACK_PATTERN='^\t\w\.go:[1-9][0-9]*(\s\+0x[0-9a-f]+)?'
-env TINY_STACK_PATTERN='^\t\?\?:[0-9][0-9]*(\s\+0x[0-9a-f]+)?$'
-
 # Tiny mode
 garble -tiny -debugdir=.obf-src build
```

Comment:
Does `-debugdir` need to be passed here?

Reply:
No